### DISCUSSION TITLE
Miracle: Smite

### DISCUSSION THREAD

**Discussion by Hoppelite** [Lambosaurus/hives-and-torches#54] [Divine Action]
`Secondary`
Add 1 more dice to your next hit

**Comment by Hoppelite** [comment]
Controversial I know, making Smite a Miracle.

**Comment by Lambosaurus** [comment]
By one more dice, would that be your weapon dice? Or a fixed dice like a d6?

**Reply by Hoppelite** [reply]
I was thinking your weapon dice, so it boosts upper damage range for great weapons and boosts crit for small weapons

**Reply by Lambosaurus** [reply]
This makes clerics with great weapons very frightening. Which is probably cool.
Sick. Lets leave this as is, and see how it plays out.
